src/amplitude_experiment/cohort/cohort_download_api.py

```diff
@@ -0,0 +1,93 @@
+import time
+import logging
+import base64
+import json
+from http.client import HTTPResponse
+from typing import Optional
+
+from .cohort import Cohort
+from ..connection_pool import HTTPConnectionPool
+from ..exception import HTTPErrorResponseException, CohortTooLargeException, CohortNotModifiedException
+
+CDN_COHORT_SYNC_URL = 'https://cohort-v2.lab.amplitude.com'
```

Comment:
We need to support EUDC for this feature.

I think we should add a server zone option to the config to automatically set the different domains based on the setting. See the client-side SDKs for reference.

So two new configs,
* server zone
* cohort server url